Package.swift

```diff
@@ -83,6 +84,11 @@ let package = Package(
             name: "MessageView",
             dependencies: ["JamitFoundation"],
             path: "Modules/MessageView"
+        ),
+        .target(
```

Comment:
You have probably developed by opening the `Package.swift` file. With this workflow, however, the new folder and files are not referenced in the Xcode project file and the pod spec isn't updated.

It would probably best to just drop CocoaPods and Carthage support (and also the Xcode project file), now that SPM performs so seamlessly. But if we keep support for them, it would be needed to also update the pod spec and the Xcode project file.